### PR TITLE
HOCS-5251: Remove actionDataUuid from Docs service

### PR DESCRIPTION
**Pull request by sameerinfo2003**
JIRA Link : https://collaboration.homeoffice.gov.uk/jira/browse/HOCS-5251

Screenshot 2022-06-24 at 06 25 46

### CODE REVIEW

**Review by LiamMacP — CHANGES_REQUESTED** [review]
This needs the table column removing as well - aslong as removing from the `DocumentData` entity.

**Comment by nickmcmahon01** [comment]
> This needs the table column removing as well - along with removing the property from the `DocumentData` entity.

This could be done in a separate PR, release 1 to remove from the DTO, release 2 to remove from the table?

**Comment by LiamMacP** [comment]
Yep agreed for the table but should still be part of the ticket. Removing from the entity in this case does need to be done here. That way we can avoid breaking changes on release.

**Comment by nickmcmahon01** [comment]
can we coordinate merging this in too: https://github.com/UKHomeOffice/hocs-docs/pull/254  to avoid a double major version increment.

File: src/main/java/uk/gov/digital/ho/hocs/document/dto/CreateDocumentRequest.java [hunk 1]
```diff
@@ -6,7 +6,6 @@
 
 import java.util.UUID;
 
-@AllArgsConstructor
```
Comment (LiamMacP) [inline src/main/java/uk/gov/digital/ho/hocs/document/dto/CreateDocumentRequest.java:9]: Could have got rid of the existing constructor and kept this - I'm not overly fussed though.

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**